packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -887,7 +887,7 @@ abstract class State<T extends StatefulWidget> with Diagnosticable {
   ///
   /// This field is used by the framework when asserts are enabled to verify
   /// that [State] objects move through their lifecycle in an orderly fashion.
-  _StateLifecycle _debugLifecycleState = _StateLifecycle.created;
+  late _StateLifecycle _debugLifecycleState = _StateLifecycle.created;
```

Comment:
Nit: while it's harmless to declare fields with trivial constant initializers `late`, it also doesn't make any difference (at least in the VM). So I would suggest to add `late` only to fields with non-constant initializers.

Reply:
I went through and blindly updated the ones in framework.dart.

For something like this it'd probably be easier to not special case it, as long as it doesn't really change anything.